code/game/objects/explosion.dm

```diff
@@ -157,8 +157,7 @@
 								var/turf/throw_at = get_ranged_target_turf(I, throw_dir, throw_range)
 								I.throw_at(throw_at, throw_range, 2, no_spin = 1) //Throw it at 2 speed, this is purely visual anyway; don't spin the thrown items, it's very costly.
 */
-		var/timeofhour = TimeOfHour
-		var/took = round((start <= timeofhour) ? ((timeofhour-start)/10) : ((timeofhour-(start-36000))/10), 0.01)
+		var/took = stop_watch(watch)

```

Comment:
and this `(world.timeofday-start)/10`

Though I don't know which will be more accurate.